colorcet/__init__.py

```diff
@@ -40,12 +40,46 @@
 Some of the Glasbey sets are aliased to short names as explained in the User Guide.
 """
 
-from .version import Version
-__version__ = str(Version(fpath=__file__, archive_commit="$Format:%h$", reponame="colorcet"))
+import os
 
 from collections import OrderedDict
 from itertools import chain
 
+# Define '__version__'
+try:
```

Comment:
Maybe keep the version simple for colorcet? 

``` python
        from importlib.metadata import version as mversion, PackageNotFoundError

        try:
            __version__ = mversion("colorcet")
        except PackageNotFoundError:
            # The user is probably trying to run this without having installed
            # the package.
            __version__ = "0.0.0+unknown"
```

Reply:
No opinion here. Certainly we don't have any strong requirements on versioning for this project, since it would be extremely rare to be using a development version of it, so a simpler version than the rest of HoloViz would be fine here. No idea whether that makes maintaining things easier overall, since it would presumably now differ from other HoloViz projects, which carries a cost of its own. Up to people more heavily currently involved!

Reply:
That was a simple copy/paste from the code in Param and I prefer that. I'm also pretty sure `importlib.metadata.version` comes with a little performance penalty, though I haven't measured it.

Reply:
My only concern is that the setuptools_scm takes a significant time to load for little benefit (in this repo):
![image](https://github.com/holoviz/colorcet/assets/19758978/6e32bcbd-8191-41bc-ad98-a27ee89cc4eb)

![image](https://github.com/holoviz/colorcet/assets/19758978/4dd1ab48-e6c7-402d-941e-2fe9188ab72c)

(created with `python -X importtime -c "import colorcet" 2> log.txt && tuna log.txt`)

Reply:
Also related to get_version https://github.com/pypa/setuptools_scm/issues/1008

Reply:
I'm not quite sure why the import time of colorcet from the repo (not sdist or wheel) matters. Anyway, simplified the code in https://github.com/holoviz/colorcet/pull/120/commits/9a974a66aadd90a6b8207bc4b7178fecaba36923.